data/compliance_profiles/schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
@jeefberkey  How do you feel about switching over to [draft-07][d7] and a d7-capable validator?

draft07 was published earlier this year, and looks pretty good.  It didn't add or change the behavior of any keywords, but it introduced features we could probably use, like if-then-else conditionals and comments for schema maintainers.

The downside is that development stopped on the [`json-schema`][g1] gem and  over a year ago, so it doesn't support draft07 .   There's another gem, [`json_schemer`][g2], that does (and is the only ruby implementation currently recognized by [json-schema.org][ji]).  However, it's pretty new and the documentation is sparse.

[ji]: http://json-schema.org/implementations.html#validator-ruby
[d7]: http://json-schema.org/draft-07/json-schema-release-notes.html
[g1]: https://github.com/ruby-json-schema/json-schema
[g2]:  https://github.com/davishmcclurg/json_schemer

Reply:
**Update:** development on the [`json-schema`][g1] gem just picked up again (new maintainer).

[g1]: https://github.com/ruby-json-schema/json-schema